src/components/TodoErrors/TodoErrors.tsx

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { FormEvent, useContext } from 'react';
+import { TodosContext } from '../../TodosContext';
+
+export const TodoErrors: React.FC = () => {
+  const {
+    errorMessage,
+  } = useContext(TodosContext);
+
+  const handleOnClick = (event: FormEvent<HTMLButtonElement>) => {
+    event.currentTarget.parentElement?.classList.add('hidden');
+  };
```

Comment:
might be better to use setErrorMessage from context
```
() => setErrorMessage('')
```

Reply:
why? I am handling here only classes to hide error visually but the existence of this block I am handling with useEffect that shows and then hides this block by setting error to an empty string

Reply:
As for me, looks more clear 🤷
if you already have setErrorMessege, might be better to use it for all similar cases
In my opinion,  in React better to toggle or change class names by using state variables.

Reply:
@adv555 you might be right but I still can't get you))
I am controlling error existence in the context but in TodoErrors I just show it and handle actions with the button
Do you think that my code doesn't work with different errors or do you just mean that it would be better to do all actions with an error block in the error block?

Reply:
It's just my thoughts, for me looks a bit tricky. 
I think it works perfectly.  I approved it from the beginning 😊